.github/workflows/wikimedia-launch.yml

```
@@ -24,7 +29,7 @@ env:
   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
 
 concurrency:
-  group: wikimedia-launch-${{ github.event.inputs.partner }}
+  group: wikimedia-launch-${{ github.event.inputs.concurrency_key != '' ? github.event.inputs.concurrency_key : github.run_id }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)